src/components/CardInfo.tsx

```diff
@@ -13,14 +13,15 @@ let CardInfo = ({ tailWide, number }: { number: number; tailWide: boolean }) =>
     title: '',
     vote_average: '',
     vote_count: '',
-    overview: ''
+    overview: '',
+
   });
-  const { poster_path, original_title, title, vote_average, overview } = data;
+  const { poster_path, original_title, title, vote_average, overview, vote_count } = data;
   const srcImg = api.changeImgLinks(poster_path, 'w342');
 
   useEffect(() => {
     api.getDataById(number).then((res: any) => {
```

Comment:
rename variable to something meaningful
change component declaration to const instead of let

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/41084902/114733244-db29f480-9d4b-11eb-85c8-82ffaff8e903.png)